app/controllers/runs_controller.rb

```diff
@@ -79,7 +83,7 @@ def get_runner
   end
 
   def run_params
-    params.require(:run).permit(:runner_id, :date, :training_plan, :progress)
+    params.require(:run).permit(:runner_id, :date, :training_plan, :progress, :changed_flag)
```

Comment:
is this "changed_flag" or "customized_flag"?   trying to think though the data model variable names.